Makefile

```diff
@@ -52,7 +52,7 @@ clean:
 	rm -rf venv/ .pytest_cache/
 
 unittest:
-	$(PYTEST) src/*/tests/
+	$(PYTEST) src/*/tests/ src/*/tests/management/
```

Comment:
I think this is already covered by `src/*/tests/`

Reply:
yes I have justed tested, so I removed